### ISSUE TITLE
False positive: WORKDIR path /opt/some-app/ must be absolute

### ISSUE THREAD

**Issue by alexeyraspopov** [RedCoolBeans/dockerlint#12]

**Comment by alexeyraspopov** [comment]
Issue doesn't appear if path set without slash at the end.

**Comment by jasperla** [comment]
Could you please include the relevant Dockerfile line which fails that shouldn't?

**Comment by dsignr** [comment]
+1

```docker
RUN rm -rf $PWD ~/.cache ~/.npm
RUN mkdir -p /srv/jupyterhub/
WORKDIR /srv/jupyterhub/
```

Fails with `WORKDIR path must be absolute`

**Comment by gimler** [comment]
Also fail with `WORKDIR path "/code" must be absolute on line...`
```
WORKDIR "/code"
```

**Comment by brandon-fryslie** [comment]
Also seeing this issue with 
```Dockerfile
WORKDIR $HOME/workspace
```

**Comment by ababushkin** [comment]
👍

**Comment by darrenhaken** [comment]
@brandon-fryslie did you get a fix for `WORKDIR $HOME/workspace`?

**Comment by reinerj** [comment]
will be fixed this week

**Comment by rnagy** [comment]
@darrenhaken this got fixed already:

ENV HOME /foo
WORKDIR $HOME/workspace


INFO: Dockerfile is OK.

**Comment by darrenhaken** [comment]
@rnagy `WORKDIR ${HOME} doesn't work`.

I can confirm `$HOME` does work though

**Comment by Genovo** [comment]
vs-code stops warning about absolute path if you remove the quotes, like this:
`WORKDIR  /app`
not
`WORKDIR  "/app"`
